abi/aapcs64/aapcs64.rst

```diff
@@ -169,41 +173,49 @@ All other content in this document is at the **Release** quality level.
-|            |                    |                                        |
-|            |                    |Several formatting changes have been    |
-|            |                    |applied to the sources to fix the       |
-|            |                    |rendered page produced by github.       |
-|            |                    |                                        |
-|            |                    |No functional changes intended from the |
-|            |                    |previous release.                       |
-|            |                    |                                        |
-|            |                    |Major changes are:                      |
-|            |                    |                                        |
-|            |                    |1. New Licence_, with relative          |
-|            |                    |   explanation in `About the license`_. |
-|            |                    |                                        |
-|            |                    |2. New sections on Contributions_,      |
-|            |                    |   `Trademark notice`_, and Copyright_. |
-|            |                    |                                        |
-|            |                    |Minor change: the section `Bit-fields   |
-|            |                    |subdivision`_ has been renamed to make  |
-|            |                    |the associated implicit link target     |
-|            |                    |unique and avoid clashing with the one  |
-|            |                    |of `Bit-fields`_.                       |
-+------------+--------------------+----------------------------------------+
+.. class:: aapcs64-change-history
+
++------------+--------------------+------------------------------------------------------------------+
+| Issue      | Date               | Change                                                           |
++============+====================+==================================================================+
+| 00Bet3     | 25th November 2011 | Beta release                                                     |
++------------+--------------------+------------------------------------------------------------------+
+| 1.0        | 22nd May 2013      | First public release                                             |
++------------+--------------------+------------------------------------------------------------------+
+| 1.1-beta   | 6th November 2013  | ILP32 Beta                                                       |
++------------+--------------------+------------------------------------------------------------------+
+| 2018Q4     | 31st December 2018 | Added rules for over-aligned types                               |
++------------+--------------------+------------------------------------------------------------------+
+| 2019Q4     | 30th January 2020  | Github release with an open source license.                      |
```

Comment:
Do we overwrite the alpha versioning? we didn't do it for the beta one, maybe we should keep the alpha too in the history?

Reply:
we never released it though.

Reply:
It's up for debate what a release is I guess in this brave new world. I don't have very strong opinions on it. I assumed the alpha moniker was to flag to the community that this isn't a final release. Now that we're doing an actual release with artifacts it's not alpha. I think to have a release without artifacts will be confusing, and I don't think there are other considerations, so it seems like a sane course of action. But I'm happy to be persuaded.

That said, I think it's confusing to see docs on github with a distinct version number. Also changing hardcoded version numbers and dates isn't fun, not a good use of our time, and it's pretty error-prone. I think we should just put in template markers like __VERSION__, and run a python script through it before converting to say PDF. I couldn't find restructured text markup that was meant to be replaced outside of the document.

Reply:
OK, so, what we have here is just to keep track of the changes. It should be considered as a CHANGELOG file, so I think we should keep track of the changes version by version.

Having said that, I am happy for you to remove the `-alpha` and forget about such version in the history. We didn't do a PDF release of the alpha version...